.fixtures.yml

```diff
@@ -1,7 +1,6 @@
 fixtures:
-  repositories:
-    'stdlib':
-      repo: 'git://github.com/puppetlabs/puppetlabs-stdlib.git'
-      ref: '3.2.0'
+  forge_modules:
```

Comment:
please revert this file, as pulling repos without versions will lead to all unpredictability in testing. Add a reference to the powershell module in the same manner that stdlib is listed.